other/restrict-secrets-store-csi-driver-loglevel-cve-2023-2878/restrict-secrets-store-csi-driver-loglevel-cve-2023-2878.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: restrict-secrets-store-csi-driver-loglevel
+  annotations:
+    policies.kyverno.io/title: Prevent CVE-2023-2878
+    policies.kyverno.io/category: Security
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: Pod, Secret, CSI-Driver
+    kyverno.io/kyverno-version: 1.11.0
+spec:
+  validationFailureAction: Audit
+  rules:
+  - name: limit-secrets-store-csi-driver-loglevel
+    match:
+      any:
+      - resources:
+          kinds:
+          - Deployment
+          - DaemonSet
+    validate:
+      message: "secrets-store-csi-driver must use log level 1 or below, CVE-2023-2878."
+      pattern:
+        containers:
+          - name: "secrets-store"
+            args: "-v=0 | -v=1"
```

Comment:
This is not correct for the matching kinds.

Reply:
I do not understand, please can you provide a bit more context?

Reply:
You're matching on Deployments but the pattern isn't accurate for a Deployment, only for a Pod.